underfs/s3a/src/test/java/alluxio/underfs/s3a/S3AUnderFileSystemTest.java

```diff
@@ -244,8 +244,8 @@ public void getOperationMode() throws Exception {
   public void stripPrefixIfPresent() throws Exception {
     Assert.assertEquals("", mS3UnderFileSystem.stripPrefixIfPresent("s3a://" + BUCKET_NAME));
     Assert.assertEquals("", mS3UnderFileSystem.stripPrefixIfPresent("s3a://" + BUCKET_NAME + "/"));
-    Assert.assertEquals("test/", mS3UnderFileSystem.stripPrefixIfPresent("test"));
-    Assert.assertEquals("test/",
-        mS3UnderFileSystem.stripPrefixIfPresent("s3a://" + BUCKET_NAME + "/test/"));
```

Comment:
can we keep this test case? `"s3a://" + BUCKET_NAME + "/test/"`

Reply:
No because I don't think /test/ is actually a valid input to this function. It was becuase I thought it was a valid input that I made this mistake.

Reply:
We can't pass in directory paths to this method? I thought any s3 UFS path can be passed to this method?

Reply:
I think directories are also allowed, since a few lines above, we have `"s3a://" + BUCKET_NAME + "/"` which is a directory.